app/account/edit/page.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+
+import {
+  Heading,
+  Flex,
+  Box,
+  Container,
+  VStack,
+  useColorModeValue
+} from '@chakra-ui/react'
+import { useSearchParams } from 'next/navigation'
+import { AccountEditForm } from './components'
+
+export default function AccountEditPage() {
+  const searchParams = useSearchParams()
+  const userInfoString = searchParams.get('userInfo')
+  const userInfo = userInfoString ? JSON.parse(userInfoString) : null
+  const bg = useColorModeValue('white', 'gray.800')
+  const color = useColorModeValue('black', 'gray.300')
+
+  return (
+    <Box as="main" minH="100svh" bg={bg} color={color}>
+      <Container
+        minW={{ base: '100%', lg: 'container.sm' }}
+        display="flex"
+        flexDirection="column"
+        alignItems={{ base: 'space-around', lg: 'flex-start' }}
+        gap={{ base: '40px', lg: '60px' }}
+        pt={{ base: '20px', md: '40px' }}
+        pb={{ base: '60px', md: '70px' }}
+      >
```

Comment:
I think the bottom has too much space. This page does not have a lot of content, so could you change it to make a space of just 70px for PC and 60px for SP?